components/esb-tools/plugins/org.wso2.integrationstudio.artifact.synapse.api/src/org/wso2/integrationstudio/artifact/synapse/api/ui/wizard/SynapseAPICreationWizard.java

```diff
@@ -750,18 +770,61 @@ public static String convertJSONtoYaml(String jsonSource) throws Exception {
 
     private String getSwaggerFileAsYAML(File swaggerFile, String apiName) {
         String swaggerContent = "";
+        boolean terminateFlag = false;
         try {
             swaggerContent = new String(Files.readAllBytes(Paths.get(swaggerFile.getAbsolutePath())));
             if (FilenameUtils.getExtension(swaggerFile.getAbsolutePath()).equals("json")) {
                 swaggerContent = convertJSONtoYaml(swaggerContent);
             }
+            if (isVersionTwoYamlFile(swaggerContent)) {
+	        	Shell shell = PlatformUI.getWorkbench().getActiveWorkbenchWindow().getShell();;
```

Comment:
appears a formatting issue, better use wso2 formatting template

Reply:
This PR is reverted by: https://github.com/Tharsanan1/integration-studio/commit/ef1e82bf7abe20bfb8300d844d2533ca4ffbb6dd. Hence resolving the comment